File: KingdomCodeTool/wwwroot/Download/202503250844/Angular/shared/CLSKetQua_XQ.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { CLSKetQua_XQ } from './CLSKetQua_XQ.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class CLSKetQua_XQService extends BaseService{
    DisplayColumns001: string[] = ['Save', 'STT', 'CLSKetQua_XQ_Id', 'SoPhieu', 'CLSYeuCau_Id', 'ID_CT'];

    List: CLSKetQua_XQ[] | undefined;
    ListFilter: CLSKetQua_XQ[] | undefined;
    FormData!: CLSKetQua_XQ;
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "CLSKetQua_XQ";
    }
}

File: KingdomCodeTool/wwwroot/Download/202503250844/Service/Implement/CLSKetQua_XQService.cs
namespace Service_eHospital_DongNai_A.Implement
{
    public class CLSKetQua_XQService : BaseService<CLSKetQua_XQ, ICLSKetQua_XQRepository>
    , ICLSKetQua_XQService
    {
    private readonly ICLSKetQua_XQRepository _CLSKetQua_XQRepository;
    public CLSKetQua_XQService(ICLSKetQua_XQRepository CLSKetQua_XQRepository) : base(CLSKetQua_XQRepository)
    {
    _CLSKetQua_XQRepository = CLSKetQua_XQRepository;
    }
    }
    }

